### PR TITLE
[OP#42100]Add newly linked packages at the top with little animation

### PR DESCRIPTION
**Pull request by SwikritiT**
OP#42100: https://community.openproject.org/projects/nextcloud-integration/work_packages/42100/activity

This PR adds the newly linked work packages at the top with a little animation and a dialog stating that the work package has been added. 

https://user-images.githubusercontent.com/41103328/168972791-0ed87ac5-c519-43c6-bbc8-eed90722f9a6.mp4

### CODE REVIEW

File: src/views/ProjectsTab.vue [hunk 1]
```diff
@@ -119,7 +117,14 @@ export default {
 			this.state = STATE.LOADING
 		},
 		onSaved(data) {
-			this.workpackages.push(data)
+			this.workpackages.unshift(data)
+			this.$nextTick(() => {
+				const workpackage = document.getElementById('workpackage-' + data.id)
+				workpackage.classList.add('workpackage-transition')
```
Comment (kiranparajuli589) [inline src/views/ProjectsTab.vue:123]: also clean the animation class after the animation is finished

File: src/views/ProjectsTab.vue [hunk 2]
```diff
@@ -119,7 +117,14 @@ export default {
...
+			})
+			showSuccess(
+				t('integration_openproject', 'Work-package linked successfully!')
```
Comment (individual-it) [inline src/views/ProjectsTab.vue:126]: we are using `Work package` in https://github.com/nextcloud/integration_openproject/blob/8bc4a7fa9f6ca689cd2e1dae436816ac8d8a4b4d/src/views/ProjectsTab.vue#L160
same in OpenProject, so we should stick to that

**Review by individual-it — CHANGES_REQUESTED** [review]
looks nice IMO

**Comment by SwikritiT** [comment]
> looks nice IMO

@individual-it tested in previous nextcloud versions, works fine in them too!

**Review by individual-it — APPROVED** [review]

**Review by kiranparajuli589 — APPROVED** [review]

**Pull request merged**